import-validation-helper/README.md

```diff
@@ -0,0 +1,18 @@
+# Python API Notebooks
+
+This directory contain an iPython notebook that introduces a few tools to check
+template MCF, statistical variable MCF, and CSV files for common errors. You can click on the
+link below to run this validation through Google Colab or run the file locally
```

Comment:
"Access the the [Colab Notebook](https://colab.research.google.com/drive/11YaPJA_g7O8_zm1iim3vNBrlW8O5SO98?usp=sharing) directly or run the file locally ..."

Then you can rid the table below.